### PR TITLE
Updated some descriptions of items and added max level for scavenger

### PR DESCRIPTION
**Pull request by MisterCheezeCake**

### CODE REVIEW

File: items/REFINED_TUNGSTEN.json [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+  "nbttag": "{HideFlags:254,SkullOwner:{Id:\"5d4caed7-a042-3bd7-8e5f-2df71f4d3f5f\",Properties:{textures:[0:{Value:\"ewogICJ0aW1lc3RhbXAiIDogMTcwNzYxNzM1MTUwMSwKICAicHJvZmlsZUlkIiA6ICJiMTM1MDRmMjMxOGI0OWNjYWFkZDcyYWVhYmMyNTQ1MCIsCiAgInByb2ZpbGVOYW1lIiA6ICJUeXBrZW4iLAogICJzaWduYXR1cmVSZXF1aXJlZCIgOiB0cnVlLAogICJ0ZXh0dXJlcyIgOiB7CiAgICAiU0tJTiIgOiB7CiAgICAgICJ1cmwiIDogImh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvNTNmZDhlODdkZDkwMzJjZDcyZjZmYTJkNmI0NWU1M2YwODI3YjUyNzI2M2RhNjQ4ZTBlZTU1NTJjZGFkNmNiYyIsCiAgICAgICJtZXRhZGF0YSIgOiB7CiAgICAgICAgIm1vZGVsIiA6ICJzbGltIgogICAgICB9CiAgICB9CiAgfQp9\"}]}},display:{Lore:[0:\"§8Collection Item\",1:\"\",2:\"§5§lEPIC\"],Name:\"§5Refined Tungsten\"},ExtraAttributes:{id:\"REFINED_TUNGSTEN\"}}",
   "damage": 3,
   "lore": [
+    "§8Collection Item",
+    "",
     "§5§lEPIC"
   ],
   "internalname": "REFINED_TUNGSTEN",
-  "crafttext": "Requires: Tungsten III & HotM 7",
+  "crafttext": "Requires: Tungsten III \u0026 HotM 7",
```
Comment (CalMWolfs) [inline items/REFINED_TUNGSTEN.json:12]: Why?

Comment (MisterCheezeCake) [reply items/REFINED_TUNGSTEN.json:12]: I truly do not know, I'll revert it

Comment (MisterCheezeCake) [reply items/REFINED_TUNGSTEN.json]: Fixed, in game editor did something weird

File: items/REFINED_UMBER.json [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
+  "nbttag": "{HideFlags:254,SkullOwner:{Id:\"c12a0b0f-fae2-3c27-801f-feb53a7770c8\",Properties:{textures:[0:{Value:\"ewogICJ0aW1lc3RhbXAiIDogMTcwODkxOTg5MjQ3NywKICAicHJvZmlsZUlkIiA6ICI4ZDYwNGY0NWM0OWQ0YWE2Yjc0MjhiNTJlYzcyYjliNSIsCiAgInByb2ZpbGVOYW1lIiA6ICJTdGFyRG9ubiIsCiAgInNpZ25hdHVyZVJlcXVpcmVkIiA6IHRydWUsCiAgInRleHR1cmVzIiA6IHsKICAgICJTS0lOIiA6IHsKICAgICAgInVybCIgOiAiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS83MDFkYjFmN2NkOWRiMTkyZGYyYzBkZjJmMDExNGY2YjkyNTg2NWQwYzM0N2YyNjVlNzk0NmU5ZDY3NjIyZWI3IgogICAgfQogIH0KfQ\"}]}},display:{Lore:[0:\"§8Collection Item\",1:\"\",2:\"§5§lEPIC\"],Name:\"§5Refined Umber\"},ExtraAttributes:{id:\"REFINED_UMBER\"}}",
   "damage": 3,
   "lore": [
+    "§8Collection Item",
+    "",
     "§5§lEPIC"
   ],
   "internalname": "REFINED_UMBER",
-  "crafttext": "Requires: Umber III & HotM 7",
+  "crafttext": "Requires: Umber III \u0026 HotM 7",
```
Comment (jani270) [inline items/REFINED_UMBER.json:12]:
```suggestion
  "crafttext": "Requires: Umber III & HotM 7",
```

**Pull request merged**